src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -64,6 +64,12 @@ internal class ConnectionManager : IConnectionManager {
             _userConnections = new ConcurrentDictionary<Uri, IConnection>(userConnections);
 
             UpdateRecentConnections();
+
+            _shell.Idle += OnIdle;
+        }
+
+        private void OnIdle(object sender, EventArgs e) {
```

Comment:
As in every F5 I do it hits null ref exception accessing DTE.MainWindow. This is b/c connection manager is up and commands get created by the package and then want interactive workflow which create connection manager which wants to populate status bar.